soapui/src/main/java/com/eviware/soapui/SoapUI.java

```diff
@@ -213,6 +218,7 @@
     public final static String PRODUCT_NAME = "SoapUI";
     public static final String DEFAULT_WORKSPACE_FILE = "default-soapui-workspace.xml";
     public static final String SOAPUI_SPLASH = "SoapUI-Spashscreen.png";
+    public static final String SOAPUI_ABOUT = "/SoapUI-blank.png";
```

Comment:
You added it, but SOAPUI_SPLASH is still used in the code.

Reply:
Fixed